AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -199,21 +201,45 @@ define([
             ];
         },
 
+        /**
+         * Initalize Tabs
+         *
+         */
+        getDataMageInit: function () {
+            $('#adobe-stock-tabs').tabs({
+                active: 'series_content',
+                destination: '#adobe-stock-tabs-content',
+                shadowTabs: []
+            });
+        },
+
         /**
          * Returns series to display under the image
          *
          * @param record
          * @returns {*[]}
          */
         getSeries: function (record) {
-            if (!record.series) {
+            if (!record.series || !record.model) {
```

Comment:
it's better to process series and same model images separately (i.e. use a loop)